### PR TITLE
Updated ppL trigger, added kaon rejection

### PR DESCRIPTION
**Pull request by lauraser**

### CODE REVIEW

File: PWGCF/FemtoDream/FemtoDreamTrackSelection.h [hunk 1]
```diff
@@ -425,6 +436,13 @@ bool FemtoDreamTrackSelection::isSelectedMinimal(T const& track)
   if (nRejectNotPropagatedTracks && std::abs(dca) > 1e3) {
     return false;
   }
+  if (nRejectITSHitandTOFMissing) {
+    TrackSelection* o2PhysicsTrackSelection = new TrackSelection(getGlobalTrackSelection());
+    ;
+    o2PhysicsTrackSelection->SetRequireHitsInITSLayers(1, {0, 1, 2, 3});
+    // Uncomment only when TOF timing is solved
+    // bool itsHit = o2PhysicsTrackSelection->IsSelected(track, TrackSelection::TrackCuts::kITSHits);
+  }
```
Comment (victor-gonzalez) [inline PWGCF/FemtoDream/FemtoDreamTrackSelection.h:445]: Please, reconsider this
You cannot be creating and leaving there a track selection object per track

**Review by victor-gonzalez — CHANGES_REQUESTED** [review]
If you need a track selection object for performing detailed selections, you should have it at the task level an create it in `init`
Then you can query it on a per track level if so you want

File: PWGCF/FemtoDream/femtoDreamProducerTask.cxx [hunk 2]
```diff
@@ -276,6 +296,14 @@ struct femtoDreamProducerTask {
         if (!v0Cuts.isSelectedMinimal(col, v0, postrack, negtrack)) {
           continue;
         }
+
+        if (ConfRejectITSHitandTOFMissing) {
+          o2PhysicsTrackSelection->SetRequireHitsInITSLayers(1, {0, 1, 2, 3});
```
Comment (victor-gonzalez) [inline PWGCF/FemtoDream/femtoDreamProducerTask.cxx:301]: In the next iteration move this to `init` and you save a function call per track

**Review by victor-gonzalez — APPROVED** [review]
I approve it now but consider my comment for the next iteration

**Review by saganatt — APPROVED** [review]

**Pull request closed without merging**